docs/modules/bindings-specification/pages/message-passing-api.adoc

```diff
@@ -189,7 +192,17 @@ class Checksums {
   /// The sha-256 checksum of this dependency's metadata.
   sha256: String
 }
+
+class Http {
+  /// Certificates to trust when making HTTP requests.
+  ///
+  /// If [null], Pkl will trust its own built-in certificates.
+  caCertificates: Binary?
```

Comment:
If I'm reading this right, this is expected to be binary data containing PEM data. Am I right? I think it might make sense for that to be in the specification.

Reply:
Yup, good call, will update